tsconfig.json

```diff
@@ -10,13 +9,12 @@
     "noEmit": true,
     "outDir": "dist",
     "resolveJsonModule": true,
+    "strict": true,
     "target": "ES2020",
-
     // Allows us to not have to typeguard in catches.
     // https://bobbyhadz.com/blog/typescript-object-is-of-type-unknown
     "useUnknownInCatchVariables": false,
-
-    "strict": true
+    "verbatimModuleSyntax": true
```

Comment:
haven't seen this one before, what's it do?

Reply:
we should probably enable it everywhere we can. it ensures that type imports use `import type SomeType` and not `import SomeType`. as always mr. pocock has a good write-up on the subject in his cheat sheet: https://www.totaltypescript.com/tsconfig-cheat-sheet

Reply:
oh cool